### ISSUE TITLE
[Bug]: The app doesn't show category descriptions

### ISSUE THREAD

**Issue by prototyperspective** [commons-app/apps-android-commons#5621]
### Summary

The WMC Android app doesn't show category description texts on category pages even if they are set on the category and are viewable on the desktop site. This is important information that should not be omitted for mobile users.

### Steps to reproduce

1. Open any category by tapping a photo in the WMC Android app and then on any of the image's categories

### Expected behaviour

It should show the category description. If a user language is set in the app configs, it should only show the category description of that language if it is available. If it is not available, the English category description should be shown.

(It could also be made possible to have both the English-language description and that of at least one additional language be shown according with preferences.) Things like hatnotes / templates should also be shown but that could become a separate issue, for now this bug report is only about category descriptions that can be viewed on desktop not being shown in the mobile app. Of these templates the See also and For templates are of highest importance and should also be made to show on mobile asap. I think this issues also includes showing infoboxes which should be collapsed by default so that the user can show the structured data with a tap on a button somewhere at the top of the page.

Improving the WMC app to a reasonable standard is high-impact / of high importance since most readers probably use WMC via smartphone by now and many of these either use the app or could be while the current quality is quite low (lots of potential). Tech development is udnerfunded with the millions of revenue per year and a campaign banner could get more volunteer devs involved without any costs. Not showing category descriptions can lead to confusion as well as lots of media in false categories. Usually category titles are self-explanatory but often they aren't which is where the descriptions are important.

### Actual behaviour

No category description is shown on the category page even if there is one set in the desktop site.

### Android version

Android (a new version)

### Commons app version

4.2.1

### Would you like to work on the issue?

Prefer not

**Comment by nicolas-raoul** [comment]
Great idea!

**Comment by mshdabiola** [comment]
Can work on it

**Comment by mshdabiola** [comment]
![Screenshot_20240327-185113.png](https://github.com/commons-app/apps-android-commons/assets/41789315/0d4a71c3-9ecb-40b4-96db-ee21e91a489a)

Can I add forth tab and name it "Description" if the category have descriptions it will display it

**Comment by nicolas-raoul** [comment]
@mshdabiola Yes a fourth tab for the description is a great idea!

**Comment by prototyperspective** [comment]
Yes a fourth tab for cat descriptions would be great! Maybe there could eventually be a setting to make the text description (maybe not anything else like the see alsos) show up at the top in main/first tab "Media" (in most cases category title are fairly self-explanatory even though they still aren't machine-translated as of 2024 while descriptions for large cats often have translations).

Just a note because the text above is a bit long: it would also show see also links to other categories and if possible a button or something like it to show the Wikidata Infobox for the category.

**Comment by sivaraam** [comment]
@mshdabiola Were you able to make any progress on this?

**Comment by nicolas-raoul** [comment]
@mshdabiola  I unassign for now, but if you are you still working on this, please let us know. If no answer, someone else may be assigned to it. Thanks a lot. :-)
